archive/index.html

```diff
@@ -29,13 +29,13 @@
 <div class="row" align="center" style="width: 100%;">
   <div class="12u pagination">
     {% if paginator.previous_page %}
-      <a href="{{ paginator.previous_page_path }}" class="previous">Previous</a>
+      <a href="{{ paginator.previous_page_path | prepend: site.baseurl }}" class="previous">Previous</a>
```

Comment:
Because the pagination links seem to be generated with the full URL and do not respect the HTML <base> tag.